templates/pipeline.jinja.html

```diff
@@ -773,12 +773,21 @@ <h1>
 
 
           {% if job["stdout"] %}
-          <div class="output-box"><xmp>stdout:
-            {% for line in job["stdout"] %}
-{{ line.strip() }}
-            {%- endfor %}{# line in job["stdout"] #}
-            </xmp>
-          </div><!-- class="output-box" -->
+            {% if job["wrapper_arguments"]["tags"] is not none and "literal-stdout" in job["wrapper_arguments"]["tags"] %}
+            <div class="pseudo">
+              {% for line in job["stdout"] %}
+  {{ line.strip() }}
+              {%- endfor %}{# line in job["stdout"] #}
+            </div><!-- class="pseudo" -->
+            {% endif %}{# literal-stdout is in tags #}
+            {% if job["wrapper_arguments"]["tags"] is none %}
```

Comment:
I don't think this is correct. There are other unrelated tags that we already use, e.g. `stats-group:xxx` (have a look in the starter kit to see what that's for).

I think it should just be `else`.